.github/workflows/meilisearch-docs-scraper.yml

```diff
@@ -65,7 +69,9 @@ jobs:
                 "autofix": ["autofix", "automatic fixes", "remediation", "code fixes"],
                 "ci": ["ci", "continuous integration", "pipeline", "github actions", "gitlab ci", "automation"],
                 "config": ["config", "configuration", "settings", "setup"],
-                "taint mode": ["taint mode", "taint tracking"]
+                "taint mode": ["taint mode", "taint tracking"],
```

Comment:
Does it make sense to add "taint" here? It looks like this may be the case based on surrounding context, but I don't know